interface/main/tabs/fullscreen.php

```diff
@@ -0,0 +1,81 @@
+<?php
+Use Esign\Api;
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+$fake_register_globals=false;
+$sanitize_all_escapes=true;
+
+/* Include our required headers */
+require_once("../../globals.php");
+require_once $GLOBALS['srcdir'].'/headers.inc.php';
+require_once $GLOBALS['srcdir'].'/ESign/Api.php';
+/* for getPnotesByUser(). */
+require_once($GLOBALS['srcdir'] . '/pnotes.inc');
+$esignApi = new Api();
+
+?>
+<!DOCTYPE html>
+<title><?php echo $GLOBALS['libreehr_name'];?></title>
+<script type="text/javascript">
+<?php require($GLOBALS['srcdir'] . "/restoreSession.php"); ?>
+
+//  Include this variable for backward compatibility
+var loadedFrameCount = 0;
+var tab_mode=true;
+function allFramesLoaded() {
+// Stub function for backward compatibility with frame race condition mechanism
+ return true;
+}
+
+function isEncounterLocked( encounterId ) {

```

Comment:
Is this function needed?

Reply:
I wasn't sure about esign and the other functions and what they did so I left them in case it was important. I'll remove them now.